File: prime/prime.java
class prime
{
    public void main(int a)
    {
        int c = 0;
        for (int i = 1; i<=a; i++)
        {
            if (a % i == 0)
            {
                c++;
            }
        }
        if (c == 2)
        {
            System.out.println("It is a prime number");
        }
        else
        {
            System.out.println("It is not a prime number");
        }
    }
}

File: prime/prime.php
<?php
// A file to print prime numbers 
function ($limit)
{
    for ($i=1; $i<=$limit, $i++)
    {
        if ($limit % i == 0)
        {
            $c = c+1;
        }
    }
    if ($c > 2)
    {
        echo "Not a prime number";
    }
    else
    {
        echo "It is a prime number";
    }
}